src/Data/SemVer/Types.hs

```diff
@@ -150,6 +150,16 @@ versionsOf = \case
   And svr1 svr2 -> versionsOf svr1 <> versionsOf svr2
   Or svr1 svr2 -> versionsOf svr1 <> versionsOf svr2
 
+stripRangeTags :: SemVerRange -> SemVerRange
```

Comment:
Although it's pretty clear what it's doing, let's drop a doc comment on this guy